client/src/features/action/needs/NeedFormGoods.tsx

```diff
@@ -69,6 +70,24 @@ function NeedForm() {
     }));
   };
 
+  const handleSubmit = async (evt: React.FormEvent) => {
+    evt.preventDefault();
```

Comment:
i don't think you need the preventDefault() and therefore not the evt arg either